utils/utils.go

```diff
@@ -41,3 +42,15 @@ func AssertError(wantError bool, err error) bool {
 	}
 	return true
 }
+
+func SShHostSplit(hostname string) (user, host string) {
```

Comment:
Consider split into file_utils.go and test_utils.go.

Is SShHostSplit being used?